.yarnrc

```diff
@@ -1,5 +1,4 @@
 --install.frozen-lockfile true
-workspaces-experimental true
```

Comment:
This was needed for yarn 0.x, so given we only use 1.x now and I'd be surprised if anyone still used 0.x this seems fine.